docs/src/userguide/pipeline/batch_conversion.rst

```diff
@@ -0,0 +1,67 @@
+.. _batch-conversion:
```

Comment:
I was thinking userguide->pipeline made sense because it was pipeline changes but agreed - this is similar to switching between fnapi and non-fnapi so i will put it there.